src/bin/sccache-dist/main.rs

```diff
@@ -73,6 +73,8 @@ enum BuilderType {
 
 enum Void {}
 
+// Only supported on x86_64 Linux machines
+#[cfg(all(target_os = "linux", target_arch = "x86_64"))]
```

Comment:
Is there a way to control this at the cargo level so that we don't build it on other platforms?